articles/azure-cache-for-redis/TOC.yml

```
@@ -11,7 +11,7 @@
   items:
   - name: Create a cache
     items: 
-    - name: Basic, Standard and Premium tiers
+    - name: Create Azure Cahce for Redis
```

Fair enough suggestion can we combine for clarity. There are also Enterprise and Enterprise Flash tier caches which use  a different procedure.
```suggestion
    - name: Create Azure Cache for Redis in the Basic, Standard and Premium tiers
```